src/services/CasperServiceByJsonRPC.ts

```diff
@@ -303,36 +303,46 @@ export class CasperServiceByJsonRPC {
   /**
    * Get information about a deploy using its hexadecimal hash
    * @param deployHashBase16 The base-16 hash of the deploy
+   * @param timeout timeout for the request
    * @returns A `Promise` that resolves to a `GetDeployResult`
    */
   public async getDeployInfo(
-    deployHashBase16: string
+    deployHashBase16: string,
+    timeout?: number
```

Comment:
can we have here an object like 
`{ timeout?: value }`? That way it will be easier to expend in future.